Date/Span.php

```diff
@@ -174,7 +174,7 @@ class Date_Span
      * @access   public
      * @see      set()
      */
-    function Date_Span($time = 0, $format = null)
+    function __construct($time = 0, $format = null)
```

Comment:
It's the same problem and the solution I mention in the previous comment thread.